administration/src/bp-modules/statistics/components/StatisticsFilterBar.tsx

```diff
@@ -0,0 +1,102 @@
+import { Button, FormGroup, Tooltip } from '@blueprintjs/core'
+import { TextField } from '@mui/material'
+import React, { ReactElement, useState } from 'react'
+import styled from 'styled-components'
+
+import PlainDate from '../../../util/PlainDate'
+import StickyBottomBar from '../../StickyBottomBar'
+import { defaultEndDate, defaultStartDate } from '../constants'
+
+const StyledFormGroup = styled(FormGroup)`
+  margin: 0 16px 0 0;
+  align-self: center;
+`
+
+const InputContainer = styled.div`
+  flex-direction: row;
+  display: flex;
+  justify-content: flex-start;
+  flex: 1;
+  padding: 0 20px;
+`
+type StatisticsFilterBarProps = {
+  onApplyFilter: (dateStart: string, dateEnd: string) => void
+}
+
+const isValidDateString = (value: string): boolean => {
+  if (value !== null) {
+    try {
+      PlainDate.from(value)
+      return true
+    } catch (error) {
+      console.error(`Could not parse date from string '${value}'.`, error)
+      return false
+    }
+  }
+  return false
+}
+const IsValidDateTimePeriod = (dateStart: string, dateEnd: string): boolean => {
+  if (!isValidDateString(dateStart) || !isValidDateString(dateEnd)) {
+    return false
+  }
+  return PlainDate.compare(PlainDate.from(dateStart), PlainDate.from(dateEnd)) === -1
+}
+
+const StatisticsFilterBar = ({ onApplyFilter }: StatisticsFilterBarProps): ReactElement => {
+  const [dateStart, setDateStart] = useState(defaultStartDate)
+  const [dateEnd, setDateEnd] = useState(defaultEndDate)
+
+  return (
+    <StickyBottomBar>
+      <InputContainer>
+        <StyledFormGroup label='Startzeitraum' inline>
+          <TextField
+            fullWidth
+            type='date'
```

Comment:
Might be nit-peaky, but maybe we would want to use European date format (TT/MM/JJ) instead of the American one (MM/TT/JJ)?

Reply:
I just noticed that we also use the mm/dd/yyyyy format on the application form.
So if we decide to adapt that, it might make sense to change it everywhere.
Do you think it makes sense to create a task for that, or it's too irrelevant and nobody cares?

Reply:
> I just noticed that we also use the mm/dd/yyyyy format on the application form.
> So if we decide to adapt that, it might make sense to change it everywhere.
> Do you think it makes sense to create a task for that, or it's too irrelevant and nobody cares?

The format depends on your device language / locale. (On German devices it uses dd.mm.yyyy) Therefore it's likely irrelevant (as most users' devices will be set to German.

Reply:
@michael-markl true, thanks 🙈